File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/go/ADV-04b_R.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func main() {
	fmt.Println("ADV-START")
	sc := bufio.NewScanner(os.Stdin)
	sc.Scan()
	p, _ := strconv.ParseInt(sc.Text(), 10, 32)
	a := int32(p)
	sc.Scan()
	q, _ := strconv.ParseUint(sc.Text(), 10, 32)
	b := uint32(q)
	v := a < b
	fmt.Printf("OBS=CMP:%t\n", v)
	fmt.Println("ADV-END")
}

File: benchmark/template/programs/kotlin/adversarial/ADV-04b_R.kt
fun main() {
    println("ADV-START")
    System.out.flush()
    val s: Int = readln().trim().toInt()
    val u: UInt = readln().trim().toUInt()
    val c: Boolean = s < u
    println("OBS=CMP:$c")
    System.out.flush()
    println("ADV-END")
    System.out.flush()
}
